Examples/ML/brain.py

```diff
@@ -0,0 +1,20 @@
+from sportsreference.nfl.teams import Teams
```

Comment:
Just a general comment for this example - while I think it's a nice example of how you can use `sportsreference` to easily create machine learning applications, I think it would be helpful to provide users context as to what exactly this does if they are unfamiliar with ML. Perhaps either adding comments, or another document as to what these steps are doing would be valuable.